linera-sdk/src/contract/mod.rs

```diff
@@ -6,126 +6,115 @@
 mod conversions_from_wit;
 mod conversions_to_wit;
 mod runtime;
-pub(crate) mod wit;
+#[doc(hidden)]
+pub mod wit;
```

Comment:
Is this `pub` now intentionally?

Reply:
Yes, because the `export_contract!` macro generated by `wit-bindgen` uses the types generated in the module :/